### PR TITLE
Adding ohai hints

### PR DESCRIPTION
**Pull request by tyler-ball**
From https://github.com/test-kitchen/kitchen-ec2/pull/89 - I meant to add these a while ago and simply forgot

### CODE REVIEW

File: lib/kitchen/driver/ec2.rb [hunk 1]
```diff
@@ -341,6 +342,15 @@ def block_device_mappings
 
         bdms
       end
+
+      # TODO Update this to use the new Transport logic when that is ready
+      def create_ec2_json(state)
+        Kitchen::SSH.new(*build_ssh_args(state)) do |conn|
+          run_remote("sudo mkdir -p /etc/chef/ohai/hints;sudo touch /etc/chef/ohai/hints/ec2.json",
```
Comment (jtimberman) [inline lib/kitchen/driver/ec2.rb]: Will sudo always be available? I think we use it elsewhere but I may misremember.

Comment (tyler-ball) [inline lib/kitchen/driver/ec2.rb]: This is not an ideal fix - when we can leverage the Transport logic from the windows-guest-support branches we will move this logic there.  This is just a temporary workaround until we get access to that.

Having said that, do you think I should add a feature flag or something, so people on systems without sudo (or `touch`) aren't totally borked by this?

**Comment by fnichol** [comment]
@tyler-ball Will this need to be reworked to fit in with #109? The only thing making me wonder is the use of `Kitchen::SSH` which can be swapped for the Transport code

**Comment by tyler-ball** [comment]
Moving the changes into https://github.com/test-kitchen/kitchen-ec2/pull/109 and closing this

**Pull request closed without merging**